docs/self-managed/operational-guides/update-guide/860-to-870.md

```diff
@@ -4,7 +4,7 @@ title: Update 8.6 to 8.7
 description: "Review which adjustments must be made to migrate from Camunda 8.6.x to Camunda 8.7.0."
```

Comment:
Is this correct? (8.7.0) As we say 8.7.x for the components? I assume so, but just checking 👍